DotNetHowToSynonyms/DotNetHowToSynonyms/Program.cs

```diff
@@ -0,0 +1,345 @@
+﻿#define HowToExample
+
+using System;
+using System.Configuration;
+using System.Linq;
+using System.Threading;
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Microsoft.Spatial;
+
+namespace AzureSearch.SDKHowToSynonyms
+{
+    class Program
+    {
+        // This sample shows how to delete, create, upload documents and query an index
+        static void Main(string[] args)
+        {
+            SearchServiceClient serviceClient = CreateSearchServiceClient();
+
+            Console.WriteLine("{0}", "Deleting index...\n");
+            DeleteHotelsIndexIfExists(serviceClient);
+
+            Console.WriteLine("{0}", "Creating index...\n");
+            CreateHotelsIndex(serviceClient);
+
+            ISearchIndexClient indexClient = serviceClient.Indexes.GetClient("hotels");
+
+            Console.WriteLine("{0}", "Uploading documents...\n");
+            UploadDocuments(indexClient);
+
+            ISearchIndexClient indexClientForQueries = CreateSearchIndexClient();
+
+            RunQueries(indexClientForQueries);
```

Comment:
You could probably delete RunQueries since it doesn't help tell the story of synonyms.